anchor-contributor/tests/helpers/contributor.ts

```diff
@@ -49,7 +53,7 @@ export class IccoContributor {
       .rpc();
   }
 
-  async initSale(payer: web3.Keypair, initSaleVaa: Buffer, saleTokenMint: web3.PublicKey): Promise<string> {
+  async initSale(payer: web3.Keypair, initSaleVaa: Buffer): Promise<string> {
```

Comment:
We now get `saleTokenMint` by grabbing it from the custodian's sale token account.